src/luac-cli.js

```diff
@@ -0,0 +1,430 @@
+#!/usr/bin/env node
+"use strict";
+
+global.WEB = false;
+
+const lua      = require('../node_modules/fengari/src/lua.js');
```

Comment:
yarn install dependencies in a flat structure unlike npm it seems... not sure how to go around that